frontend/src/pages/PipelineList.tsx

```diff
@@ -75,7 +75,7 @@ class PipelineList extends Page<{}, PipelineListState> {
 
   public getInitialToolbarState(): ToolbarProps {
     let buttons = new Buttons(this.props, this.refresh.bind(this));
-    if (isFeatureEnabled('v2')) {
+    if (isFeatureEnabled(FEATURE_KEY.V2)) {
       buttons.visualizeIRPipeline();
```

Comment:
nit: shall we consider using the same path for v1 and v2 pipeline details page?
The difference is,
if different path, every callsite need to know in advance whether a pipeline is v1 or v2.
If the same path, the details are hidden from callsite, we just know the pipeline I'd and navigate.

In react, it's easy to replace a component with a wrapper component that accepts the same arguments, you can detect pipeline version there, and decide to show the v1 or v2 details page.

Reply:
To make sure I understand, let me explain the current workflow:

<pre>
UI-> go PipelineList-> click one pipeline -> open PipelineDetails page as wrapper -> PipelineDetailsV1 if YAML
                                                       | 
                                                        ---------------------------> PipelineDetailsV2 if IR
</pre>


The change you highlight here is a button I added temporarily to visit PipelineDetailsV2, I can actually remove this because now I have the PipelineDetails wrapper which can open PipelineDetailsV2. 

So looks like I am already doing the wrapper component approach you suggest, am I understanding it correctly?

Reply:
Yes, the diagram is exactly what I thought.
Then all makes sense to me, thanks!

Reply:
Awesome, thank you Yuan for confirming! It makes sense to me, I will remove the v2/pipelines/details path and the IR PIPELINE button.